packages/common/src/llm/schemas.ts

```diff
@@ -29,7 +32,10 @@ const MessageSchema = z.object({
       z.array(
```

Comment:
:thought_balloon: **thought:** that `string.or.z.array` kind of bugs me, could we use a cleaner / simpler API for the generateContent methods, even if we have to do a small conversion from bp -> openai?

Reply:
This follows the same pattern as OpenAI and Anthropic where the content can simply be text, or if you want a multipart message, then you can pass an array, which I think is really nice for DX because if you're only dealing with text prompts (which I think is the most common scenario) it feels awkward and annoying to have to pass a complex content object just for a plain string?